kimchi/src/error.rs

```diff
@@ -44,6 +44,9 @@ pub enum VerifyError {
     #[error("the previous challenges have an unexpected length (expected {0}, got {1})")]
     IncorrectPrevChallengesLength(usize, usize),
 
+    #[error("the evaluation is of an unexpected size")]
```

Comment:
nit: a bit vague, perhaps a better error is "proof malformed: an evaluation was of the incorrect size (all evaluations are expected to be of length 1)"